src/main.ts

```diff
@@ -51,11 +51,13 @@ async function run() {
             userFractionFloat = undefined
         }
 
-        // Check release file
-        if (!fs.existsSync(releaseFile)) {
-            core.setFailed(`Unable to find release file @ ${releaseFile}`);
-            return
-        }
+        // Check release files
+        releaseFiles.forEach(releaseFile => {
```

Comment:
Please convert this to a `for (in)` loop since the keyword `return` doesn't break a `forEach()` loop